src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, AreaCalculable {
```

Comment:
read about why we need `abstract`
```suggestion
public abstract class Figure implements Drawable, AreaCalculable {
```